ext/data/templates/default-anki-field-templates.handlebars

```diff
@@ -302,6 +302,54 @@
     {{~/if~}}
 {{/inline}}
 
+{{#*inline "frequencies-num"}}
```

Comment:
Avoid abbreviating the name, `frequencies-number` or `frequency-number` is fine.

Reply:
(If this is outputting a single number, prefer the singular.)

Reply:
Good suggestions, fixed this.